tools/main.cpp

```diff
@@ -57,7 +57,7 @@ int main(int argc, char** argv) {
    app.add_flag_callback("-V,--version",
          [&app]() {
             std::cout << app.get_name() << " v" << antler::system::version::full() << std::endl;
-            std::exit(0);       // Succesfull exit MUST happen here.

```

Comment:
now I feel relief that native English speakers also mess double letters in that word lol